craftninja/practice-exp-app
# Add standard goodies

### What does this PR do?

Adds some standard goodies such as:
* PR template
* Circle CI testing
* local test coverage tool nyc
* Code Climate code review and test coverage

### Where should someone start to review?

that circle file has most of it 😬 

### How could you test this PR's functionality?
* merge PR and see new PR template for new PRs
* View CircleCI tests passing
* Pull down branch and run `yarn coverage` and then `yarn report` following instructions in README
* View CodeClimate test coverage

### New dependencies?

[nyc](https://www.npmjs.com/package/nyc)

### Gif of your feels:

![bun](https://user-images.githubusercontent.com/6353499/66352837-d34ef000-e91d-11e9-958d-9ef0a2a560d6.gif)
